cli/commands/console.go

```diff
@@ -175,3 +175,11 @@ func (o *ConsoleOptions) Run() error {
 
 	return console.Serve(serveOpts)
 }
+
+func (o *ConsoleOptions) addressOrAPIHost() string {
```

Comment:
@sho-hata On a second thought, We can give `--api-host` the priority when both flags are set. When someone set the flags they probably want to set the value of "--api-host" different from `--address`.

Reply:
@scriptonist OK, We will modify the code to give priority to the value of `--api-host` when both options are specified.

Reply:
@scriptonist I fixed the behavior, please check it out.
https://github.com/hasura/graphql-engine/pull/8039/commits/7ea85f09434b63bb14bdcae4b3ef5c6e6116e4d9

Reply:
Thanks @sho-hata will check it out and get back.